.github/CODEOWNERS

```diff
@@ -1023,6 +1024,7 @@ x-pack/solutions/search/plugins/search_query_rules @elastic/search-kibana
 x-pack/solutions/search/plugins/search_solution/search_navigation @elastic/search-kibana
 x-pack/solutions/search/plugins/search_synonyms @elastic/search-kibana
 x-pack/solutions/search/plugins/serverless_search @elastic/search-kibana
+x-pack/solutions/search/test
```

Comment:
should this have @elastic/search-kibana as the code owner?

Reply:
should be added by CI in 149360fbfffbc242961e7db82e17e6f7961ab1b2